source/asdmanager.py

```diff
@@ -113,9 +113,14 @@ def setup():
         print Interactive.boxed_message(['API port cannot be in the range of the ASD port + 100'])
         sys.exit(1)
 
-    store = Interactive.ask_choice(['Arakoon', 'Etcd'],
-                                   question='Select the configuration management system',
-                                   default_value='Arakoon').lower()
+    if run_interactive is False:
+        store = config['asdmanager'].get('store')
```

Comment:
Add a ```.lower()``` here too, because interactively customer's asked for 'Arakoon' or 'Etcd', so he might use this in the preconfig too, but this would fail